### ISSUE TITLE
[BUG] blank / message screen after update

### ISSUE THREAD

**Issue by ylluminate** [Lightcord/Lightcord#230]
Running macOS 10.15.7; was working fine prior to update with 1.4, but after update I only get these screens with messages in them and a throbbing "B" in the lower right corner:
small screencast: https://p153.p0.n0.cdn.getcloudapp.com/items/qGuEwQZr/060d3211-4ee5-4a96-bf3f-f2d9b017916f.gif

![screencast](https://p153.p0.n0.cdn.getcloudapp.com/items/qGuEwQZr/060d3211-4ee5-4a96-bf3f-f2d9b017916f.gif)

**Comment by ylluminate** [comment]
Same as this: https://github.com/Lightcord/Lightcord/issues/171

Closed this duplicate in favor of previous.

**Comment by AvaQueen-Official** [comment]
Interesting, how can we get version 1.3 without it updating to 1.5?

**Comment by ylluminate** [comment]
This has been very frustrating - have had to stop using Lightcord since the developer appears to ignore issues (as they have seemingly acknowledged they do). I guess someone needs to fork the project or start making nightly builds via some CI workflow.

**Comment by AvaQueen-Official** [comment]
Oh I see. Do you happen to have a copy of the older version that works per chance?

**Comment by GermanBread** [comment]
> This has been very frustrating - have had to stop using Lightcord since the developer appears to ignore issues (as they have seemingly acknowledged they do). I guess someone needs to fork the project or start making nightly builds via some CI workflow.

macos support is flakey.

The main devs don't have a mac to properly support macos builds

**Comment by smartfrigde** [comment]
Tl;Dr if you are using Mac OS Lightcord isn't for you. Just install something as simple as normal client mod.

**Comment by AvaQueen-Official** [comment]
@GermanBread @smartfrigde Got it. Is there anything that can make BetterDiscord less bloated and slow...or an alternative speedier betterdiscord? I have it installed with like 5 plugins and everything is significantly slower.

**Comment by smartfrigde** [comment]
Just use GooseMod, Powercord or any other alternative

**Comment by AvaQueen-Official** [comment]
Ok will do!
